operator/v1/types_ingress.go

```diff
@@ -344,6 +344,52 @@ type IngressControllerSpec struct {
 	// +kubebuilder:default:="Immediate"
 	// +default="Immediate"
 	IdleConnectionTerminationPolicy IngressControllerConnectionTerminationPolicy `json:"idleConnectionTerminationPolicy,omitempty"`
+
+	// closedClientConnectionPolicy controls how the IngressController
+	// behaves when the client closes the TCP connection while the TLS
+	// handshake or HTTP request is in progress. This option maps directly
+	// to HAProxy’s "abortonclose" option
+	//
+	// Allowed values for this field are "Abort" and "Continue".
+	// The default value is "Continue".
+	//
+	// When set to "Abort", the router will stop processing the TLS handshake
+	// if it is in progress, and it will not send an HTTP request to the backend server
+	// if the request has not yet been sent when the client closes the connection.
+	//
+	// When this option is set to "Abort", the router will stop processing
+	// the TLS handshake if it is in progress and will not send an HTTP
+	// request to the backend server if it has not yet been sent when the
+	// client closes the connection.
+	//
+	// When set to "Continue", the router will complete the TLS handshake
+	// if it is in progress, or send an HTTP request to the backend server
+	// and wait for the backend server's response, regardless of
+	// whether the client has closed the connection
+	//
+	// Setting "Abort" can help free CPU resources otherwise spent on TLS computation
+	// for connections the client has already closed, and can reduce request queue
+	// size, thereby reducing the load on saturated backend servers.
+	//
+	// Important Considerations:
+	//
+	//   - The default policy ("Continue") is HTTP-compliant and requests
+	//     for aborted client connections will still be served.
+	//     Prefer using the Continue" policy when a client sends a request
+	//     and then immediately closes its side of the connection while
+	//     still expecting a response on the half-closed connection
```

Comment:
You're missing a double-quote (I missed that before) and a period.  Also, it might be clearer to this paragraph as, "Use X to to allow Y".
```suggestion
	//     Use the "Continue" policy to allow a client to send a request
	//     and then immediately close its side of the connection while
	//     still receiving a response on the half-closed connection.
```